index.html

```diff
@@ -38,6 +38,7 @@
 
   <!-- LineIcons -->
 
+  <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
```

Comment:
Can you add this link tag in a new comment like below @Devemmy01 
````html
<!-- Boxicons -->

<link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
````

Reply:
I'll make all changes now